src/Tabs/Tab.js

```diff
@@ -0,0 +1,109 @@
+import React, { forwardRef } from 'react'
+import PropTypes from 'prop-types'
+import cx from 'classnames'
+import { colors } from '../theme.js'
+
+const noop = () => {}
+
+const Tab = forwardRef(
+    ({ icon, onClick, selected, disabled, fixed, children }, ref) => (
+        <button
+            className={`${cx({
+                selected,
+                disabled,
+                fixed,
+            })}`}
+            onClick={disabled ? noop : onClick}
+            ref={ref}
+        >
+            {icon && icon}
```

Comment:
Should be `{icon}`.
If the left-hand value is falsy, this expression will return the value of the left-hand variable, which is empty string.